test/integration/connect/envoy/README.me

```diff
@@ -0,0 +1,14 @@
+# Envoy Integration Tests
+
+## Overview
+
+These tests validate Envoy configuration. They set up various scenarios using Docker containers and then run
+[Bats](https://github.com/sstephenson/bats) tests (a Bash test framework) to validate the expected results.
+
+## Running Tests
+
+To run the tests locally, `cd` into the root of the repo and run:
+
+```console
+make
```

Comment:
```suggestion
make test-envoy-integ GO_TEST_FLAGS="-run TestEnvoy/<test-case>"
```

Reply:
thanks sorry I meant to include this file in a separate PR.